cmd/containerd/command/service_windows.go

```diff
@@ -124,6 +137,22 @@ func getServicePath() (string, error) {
 	return filepath.Abs(p)
 }
 
+// getPriorityValue returns the value associated with a Windows process priorityClass
+// Ref: https://docs.microsoft.com/en-us/windows/win32/cimwin32prov/setpriority-method-in-class-win32-process
+// TODO: Think of a better way to share code between kubelet and kube-proxy as this function definition is same as what
+// 		 we have cmd/kubelet/app
+func getPriorityValue(priorityClassName string) uint32 {
+	var priorityClassMap = map[string]uint32{
+		"IDLE_PRIORITY_CLASS":         uint32(64),
```

Comment:
We should use the constants defined in `golang.org/x/sys/windows` ([here](https://pkg.go.dev/golang.org/x/sys/windows#ABOVE_NORMAL_PRIORITY_CLASS)) instead of numeric literals.

Reply:
I think they have to be explicitly converted to uint32 as the SetPriority expects uint32 as the argument. If you prefer the using the values in the golang, I am fine changing this method to a hex to uint32 converter method and create a map from there.

Reply:
I think it's fine to convert to uint32 here, but we should use the already defined values as it improves readability.

e.g.:
```
var priorityClassMap = map[string]uint32{
    "IDLE_PRIORITY_CLASS": uint32(IDLE_PRIORITY_CLASS),
    ...
}
```